package com.project.academia_api.domain.exception;

public class CpfJaCadastradoException extends RuntimeException{
    public CpfJaCadastradoException(String cpf) {
        super("JÃ¡ existe um aluno cadastrado com o CPF: " + cpf);
    }
}
